.github/workflows/ci.yaml

```diff
@@ -58,7 +58,7 @@ jobs:
       - name: Run pytest
         run: pytest -v
 
-      - if: runner.os != 'macOS'
+      - if: matrix.os != 'macos-latest'
```

Comment:
> matrix.os is easier to understand as it is a direct comparison with
> the values provided in the matrix definition.

Not a bit deal, but the downside is that this will now have to be updated if/when we add other macOS versions to the matrix. I bet we'll forget… ;-) My use of `runner.os` here was intentional for this reason.

Reply:
I see `'macOS'` as an additional value to look up in documentation. But this change isn't necessary so I'll revert.